README.md

```diff
@@ -11,11 +11,16 @@ The ODF CLI tool provides configuration and troubleshooting commands for OpenShi
   - `recovery-profile`: Get the recovery profile value.
   - `health`: Check health of the cluster and common configuration issues.
 - `odf purge-osd <ID>`: Permanently remove an OSD from the cluster.
-- `odf help` : Display help text
-- `odf subvolume ls`: Display all the subvolumes
-- `odf subvolume delete <subvolume> <filesystem> <subvolumegroup>`: Deletes the stale subvolumes
+- `odf maintenance`: [Perform maintenance operations](docs/maintenance.md) on mons or OSDs. The mon or OSD deployment will be scaled down and replaced temporarily by a maintenance deployment.
```

Comment:
maintenance is more general, how about troubleshoot?

Reply:
we already discussed and agreed with `maintenance` also the upstream PR is merged with `maintenance`. Sorry but you are a little late in the discussion:XD